sync/downstream.py

```diff
@@ -245,6 +245,14 @@ def skip(self) -> bool:
     def skip(self, value: bool) -> None:
         self.data["skip"] = value
 
+    def try_rebase(self) -> DownstreamAction:
+        try:
+            logger.info("Rebasing onto %s" % self.gecko_integration_branch())
```

Comment:
I wonder if we're better off rebasing onto the integration branch (i.e. autoland) or onto central? Maybe the thing to do would be to check if central's been updated and if so rebase onto the new central, and if not keep the existing behaviour.

Reply:
I think with this I'll need your help. I'm not sure, how to check that central it's updated